cpp/src/parquet/bloom_filter_benchmark.cc

```diff
@@ -0,0 +1,230 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "benchmark/benchmark.h"
+
+#include "parquet/bloom_filter.h"
+
+#include <random>
+
+namespace parquet {
+namespace benchmark {
+
+constexpr static uint32_t kBloomFilterElementSize = 1024;
```

Comment:
I'm curious, is this a typical bloom filter size? Given that bloom filters are grouped by row group, I would perhaps expect a larger value? (row groups are typically quite large)

Reply:
Also, let's call this `kNumBloomFilterInserts` or something?

Reply:
> I'm curious, is this a typical bloom filter size? Given that bloom filters are grouped by row group, I would perhaps expect a larger value? (row groups are typically quite large)

It's an element size, maybe I can make it 10000, because our system default using 10000 as row-group stride?

Reply:
Actually, `DEFAULT_MAX_ROW_GROUP_LENGTH` is `1024 * 1024` currently.

Reply:
Emmm should I use 1mib rows here? Seems it would be extremly huge

Reply:
I don't know, is it not a typical row group size? Let's see if it makes the benchmarks too slow.

Reply:
Okay